src/ontology/analysis/analyze_mondo_stats/README.md

```diff
@@ -0,0 +1,307 @@
+```
+
+
+#### Alignment to the source NCIT - Number of terms from the sources that are in Mondo (as Mondo:equivalent)
+- Query Mondo for a _list_ of all Mondo classes that are in the human disease branch, i.e. children of MONDO:0700096 'human disease' and have an xref to NCIT and also have an xref source annotation of `MONDO:equivalentTo`. Run from `mondo/src/ontology/analysis/analyze_mondo_stats` as: 
+```
+robot reason -i ../../mondo-edit.obo query --use-graphs true -q sparql/get_mondo_class_list_with_ncit_xref-equivalentTo.ru data/input/all_mondo_classes_with_ncit_xref-equivalentTo.tsv
+```
+
+- Check of overlapping NCIT CURIEs in each file by running these commands from the Terminal:
+```
+$ awk -F'\t' 'NR==FNR {xref[$1]; next} $3 in xref' data/input/ncit_neoplasm_children.tsv data/input/all_mondo_classes_with_ncit_xref-equivalentTo.tsv > data/output/all_mondo_classes_with_ncit_xref_neoplasm-branch-equivalentTo.tsv
+
+$ wc data/output/all_mondo_classes_with_ncit_xref_neoplasm-branch-equivalentTo.tsv
+    3452   18507  210236 data/output/all_mondo_classes_with_ncit_xref_neoplasm-branch-equivalentTo.tsv
+```
+
+#### Results for Alignment with the source - Number of terms from the sources that are in Mondo (as Mondo:equivalent)
+- DOID	11185
+- ICD10CM	1116
+- NCIT	3452
+- OMIM	9141
+- Orphanet	8176
+- UMLS	20738
+
+
+### Request 3: Externally Managed Content
+For Mondo terms are in the human disease branch, i.e. children of MONDO:0700096 'human disease', get the total number of diseases represented by each Externally Managed Content source (see below). This should be the count of distinct Mondo classes that contain at least one database_cross_reference to each of the "Externally Managed Content" sources. The query can be run against `mondo-edit.obo`, but must use the parameter `reason` with the robot command since many Mondo classes only classify with a parent of MONDO:0700096 'human disease' in the inferred hierarchy.
+
+- Run from `mondo/src/ontology/analysis/analyze_mondo_stats` as: 
+```
+robot reason -i ../../mondo-edit.obo query --use-graphs true -q sparql/get_all_human_disease_mondo_classes_with_EMC_xref.ru data/output/all_human_disease_mondo_class_xref_EMC_count.tsv
+```
+
+#### Externally Managed Content
+- GARD
+- MedGen
+- NANDO
+- NORD
+
+#### Results for Externally Managed Content
+- GARD	10719
+- MedGen	20738
+- NANDO	1499
+- NORD	911
+
+
+### Request 4 - General Statistics
+These can be generated as: `sh run.sh make create-mondo-stats`.
+
+#### Results - General Statistics
+All Mondo Stats created on: Tue Jan 14 05:02:59 UTC 2025
+?classCount
+25470
+?xrefCount
+127289
+?clsDefinitionCount
+17550
+?exactSynonymCount
+73357
+?relatedSynonymCount
+30474
+?narrowSynonymCount
+2539
+?broadSynonymCount
+1385
+?rareClassCount
+15641
+?infectiousClassCount
+1071
```

Comment:
how was infectious class counted?

Reply:
children of `MONDO_0005550`. I updated the README to highlight where these queries can be found.